README.md

```diff
@@ -307,7 +307,7 @@ account).
 
 The main venue for distribution of Python software is the [Python Package Index](https://pypi.python.org/), or PyPI, also lovingly known as "the cheese-shop".
 
-To distribute your software on PyPI, you will need to create a user account on PyPI. You can upload your software using `python setup.py upload`.
+To distribute your software on PyPI, you will need to create a user account on [PyPI](http://python-packaging-user-guide.readthedocs.org/en/latest/distributing/#register-your-project). It is recommended that you upload your software using [twine](http://python-packaging-user-guide.readthedocs.org/en/latest/distributing/#upload-your-distributions).
```

Comment:
Thanks for pitching in. It seems that we'd need to explain how we got to the point where we have a `dist/*` to upload, though. Arguably, that was also true before, so it's not a complaint against your proposed change, but more of a question.